From: Dara Adib <daradib@ocf.berkeley.edu>
Date: Sun, 31 May 2015 00:01:24 -0700
Subject: fix linked libs

Remove ext from INCLUDEPATH. This directory contains
upstream copies of third-party headers which are already
packaged in Debian.

Do not link against unused libraries:
dynamic loading, udev, zlib

Rename libquazip to libquazip-qt5 for Qt5 build.
---
 Sachesi.pro | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/Sachesi.pro b/Sachesi.pro
index b85333c..17f9560 100644
--- a/Sachesi.pro
+++ b/Sachesi.pro
@@ -8,7 +8,7 @@ VERSION = 2.0.0
 
 # Global specific
 CONFIG += c++11
-INCLUDEPATH += ext src
+INCLUDEPATH += src
 P = $$_PRO_FILE_PWD_
 
 win32 {
@@ -58,12 +58,11 @@ else:android {
     ANDROID_PACKAGE_SOURCE_DIR = $$P/Android
     DEFINES += BOOTLOADER_ACCESS
 } else {
-    shared_quazip: LIBS += -lquazip
+    shared_quazip: LIBS += -lquazip-qt5
     shared_lzo2 {
         LIBS += -llzo2
         DEFINES += _LZO2_SHARED
     }
-    LIBS += -lz -ldl -ludev
     # These below should be static for it to be fully portable (changing ABIs)
     LIBS += -lcrypto -lusb-1.0
     DEFINES += BOOTLOADER_ACCESS
